packages/edit-site/src/components/global-styles/screen-revisions/index.js

```diff
@@ -95,7 +95,7 @@ function ScreenRevisions() {
 			<ScreenHeader
 				title={ __( 'Revisions' ) }
 				description={ __(
-					'Revisions are added to the timeline when style changes are saved.'
+					'Click on previously saved styles to preview them. To restore a selected version to the editor, hit "Apply". When you are ready, save the template to commit your changes.'
```

Comment:
```suggestion
					'Click on previously saved styles to preview them. To restore a selected version to the editor, hit "Apply". When you\'re ready, use the Save button to save your changes.'
```

Sorry, my fault. Now that I re-read it, I think the words I suggested, "template" and "commit", are too jargony. Maybe replace simply with "save" as above.

What do you think? Does it still make sense?

Reply:
Totally agree. I replaced the text.